### PR TITLE
Fixed AddItem command parser bugs and small UG fix

### PR DESCRIPTION
**Pull request by Bahamas20**
Fixed bugs:
#174 
#241 
#236

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddItemCommand.java [hunk 1]
```diff
@@ -39,8 +39,13 @@ public class AddItemCommand extends Command {
     public static final String MESSAGE_DUPLICATE_SUPPLIER = "This supplyItem from same"
             + " supplier already exists in inventory";
 
-    public static final String MESSAGE_INVALID_CURRENT_STOCK = "Current Stock field must be provided";
-    public static final String MESSAGE_INVALID_MINIMUM_STOCK = "Minimum Stock field must be provided";
+    public static final String MESSAGE_MISSING_CURRENT_STOCK = "Current Stock field must be provided";
+    public static final String MESSAGE_MISSING_MINIMUM_STOCK = "Minimum Stock field must be provided";
+    public static final String MESSAGE_INVALID_STOCK = "Stock fields provided must be positive";
```
Comment (hauchongtang) [inline src/main/java/seedu/address/logic/commands/AddItemCommand.java:44]: More comprehensive handling 👍

File: src/main/java/seedu/address/logic/parser/AddItemCommandParser.java [hunk 2]
```diff
@@ -38,11 +40,30 @@ public AddItemCommand parse(String args) throws ParseException {
...
+                    AddItemCommand.MESSAGE_MISSING_MINIMUM_STOCK));
+        }
+
+        try {
+            currentStock = parseInt(currentStockValue);
+            minimumstock = parseInt(minimumStockValue);
+        } catch (NumberFormatException e) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    AddItemCommand.MESSAGE_USAGE));
+        }
```
Comment (hauchongtang) [inline src/main/java/seedu/address/logic/parser/AddItemCommandParser.java:62]: You want to handle negative as well ? I might do the same for editStock

Comment (hauchongtang) [reply src/main/java/seedu/address/logic/parser/AddItemCommandParser.java:62]: I think it was in one of the bug report #205

**Review by hauchongtang — COMMENTED** [review]
Better handling of the various regressions.

File: src/main/java/seedu/address/logic/parser/AddItemCommandParser.java (hunk 2 above)
Comment (Bahamas20) [reply src/main/java/seedu/address/logic/parser/AddItemCommandParser.java:62]: Stated below that currentstock and minimumstock should be positive already 
Should handle negative values

File: src/main/java/seedu/address/logic/parser/AddItemCommandParser.java [hunk 3]
```diff
@@ -38,11 +40,30 @@ public AddItemCommand parse(String args) throws ParseException {
...
 
         if (currentStock <= 0 || minimumstock <= 0) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddItemCommand.MESSAGE_USAGE));
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    AddItemCommand.MESSAGE_INVALID_STOCK));
```
Comment (Bahamas20) [inline src/main/java/seedu/address/logic/parser/AddItemCommandParser.java:66]: Here @hauchongtang

**Review by hauchongtang — APPROVED** [review]
LGTM !

**Pull request merged**